### PR TITLE
[Java-Integration]: Add new testcases

### PR DESCRIPTION
**Pull request by arthurscchan**
Related to step 4~6 of issue #536.

Create more testcases to verify the call graph generation process for Soot module. 

Signed-off-by: Arthur Chan <arthur.chan@adalogics.com>

### CODE REVIEW

File: tests/java/test9/Fuzz/FunctionTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+///////////////////////////////////////////////////////////////////////////
+
+package Fuzz;
+
+public class FunctionTest {
+	protected void function1() {
+		System.out.println("F1")
```
Comment (AdamKorcz) [inline tests/java/test9/Fuzz/FunctionTest.java:20]: Is the print statement used for anything in the test or is it used for debugging locally only?

File: tests/java/test8/build.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```
Comment (AdamKorcz) [inline tests/java/test8/build.sh:1]: Please add license header.

File: tests/java/test9/Fuzz/FunctionTest.java (hunk 1 above)
Comment (arthurscchan) [reply tests/java/test9/Fuzz/FunctionTest.java:20]: Just to add an regression call back to some library calls.

File: tests/java/test8/build.sh (hunk 2 above)
Comment (arthurscchan) [reply tests/java/test8/build.sh:1]: done

Comment (arthurscchan) [reply tests/java/test8/build.sh:1]: done

**Review by AdamKorcz — APPROVED** [review]

**Pull request merged**